src/rtt.zig

```diff
@@ -178,7 +179,6 @@ pub const channel = struct {
                     },
                     _ => unreachable,
                 }
-                exclusive_access.unlockFn(exclusive_access.context);
                 return bytes.len;
```

Comment:
could just be unreachable?

Reply:
could also maybe just `return switch(...)` and have each prong break with bytes written